modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,66 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function(){
+     // loading aria attribute on pates tabs
+     $('.nav-tabs').attr('role', 'tablist');         // ul
+     $('.nav-tabs li').attr('role', 'presentation'); // li
+     $('.nav-tabs a').attr('role', 'tab');           // a
```

Comment:
Why is this set in JS and not just on the HTML itself?

Reply:
They are on HTML so I will remove these.